examples/Google.Cloud.Functions.Examples.FSharpHttpFunction/Function.fs

```diff
@@ -26,5 +26,5 @@ type Function() =
         /// <returns>A task representing the asynchronous operation.</returns>
         member this.HandleAsync context =
             async {
-                context.Response.WriteAsync "Hello, Functions Framework." |> Async.AwaitTask |> ignore
+                do! context.Response.WriteAsync "Hello, Functions Framework." |> Async.AwaitTask
```

Comment:
Thanks! Much appreciated :) (I did experiment for a while trying to get this to work, but must have missed the option of `do!`.)